typescript/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "example-typescript-app",
+  "version": "1.0.0",
+  "description": "Typescript example",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "compile": "tsc --declaration",
+    "lint": "tslint --project tsconfig.json",
+    "test": "mocha"
+  },
+  "directories": {
+    "lib": "./lib",
+    "src": "./src",
+    "test": "./src"
+  },
+  "dependencies": {
+    "typescript": "^3.3.3"
```

Comment:
Project still Typescript-dependent after compile (prepublish)?
If not, devDependency...